src/MensaWidget/MWFrame.class.st

```diff
@@ -11,43 +11,160 @@ Class {
 	#'squeak_changestamp' : 'Louis Christ 5/27/2024 19:43'
 }
 
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'tmc 6/6/2024 21:40'
+}
+MWFrame class >> cellSpacing [
+
+	^ 10
+]
+
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'tmc 6/6/2024 21:36'
+}
+MWFrame class >> contentExtent [
+
+	^ 450 @ 550
+]
+
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'tmc 6/6/2024 21:03'
+}
+MWFrame class >> contentMorphColor [
+
+	^ Color darkGray
+]
+
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'Oscar 6/5/2024 00:54'
+}
+MWFrame class >> contentMorphInnerBorder [
+
+	^ 5
+]
+
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'tmc 6/6/2024 21:50'
+}
+MWFrame class >> contentPosition [
+
+	^ 5 @ 40
+]
+
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'Louis Christ 6/6/2024 19:50'
+}
+MWFrame class >> frameSize [
+
+	^ 460 @ 595
+]
+
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'Oscar 6/5/2024 01:01'
+}
+MWFrame class >> scrollBarThickness [
+
+	^ 20
+]
+
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'Oscar 6/5/2024 00:57'
+}
+MWFrame class >> spacingBetweenCanteenMorphs [
+
+	^ 5
+]
+
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'Oscar 6/5/2024 01:13'
+}
+MWFrame class >> windowHeaderSize [
+
+	^ 40 
+]
+
 {
 	#category : #creation,
-	#'squeak_changestamp' : 'Louis Christ 5/29/2024 18:43'
+	#'squeak_changestamp' : 'tmc 6/6/2024 21:37'
 }
-MWFrame >> addText [
+MWFrame >> buildFrameContent [
 
-	| textMorph |
+	| contentMorph |
 	
-	textMorph := TextMorph new.
-	textMorph contents: 'Are you a hungry?';
-		position:  135@100.
-	self addPaneMorph: textMorph.
+	contentMorph := ScrollPane new.
+	self fillContentMorph: contentMorph.
```

Comment:
i think when naming methods, we should not double the variable. e.g. imo fill: aContentMorph would be better than fillContenMorph: aContentMorph. The method name then would be more generic, yes, but you can always look it up and see that in fill: aContentMorph or fill: aMorph it expects a Morph.